packages/rpc-graphql/src/resolvers/block.ts

```diff
@@ -23,7 +23,7 @@ export const resolveBlock = (fieldName?: string) => {
     ) => {
         const slot = fieldName ? parent[fieldName] : args.slot;
 
-        if (slot) {
+        if (slot != undefined) {
             if (onlyFieldsRequested(['slot'], info)) {
                 return { slot };
             }
```

Comment:
👍

Reply:
How does this behave if I do:

```graphql
{
    block: {
        __id
        __type
        slot
    }
}
```

Relay, for instance, basically adds `__id` and `__type` to every subselection automatically.

Reply:
Yeah GraphQL-JS adds those!

Reply:
What I mean to ask is if this resolver will get deopted (ie. make the fetch) if the client asks for `__id` and `__type`.

Reply:
Ohhh yeah it might actually. I'll add a PR for this. Good call.